src/mcts/search.cc

```diff
@@ -519,7 +519,6 @@ void Search::RunBlocking(size_t threads) {
 }
 
 bool Search::IsSearchActive() const {
-  Mutex::Lock lock(counters_mutex_);
   return !stop_;
```

Comment:
`!stop_.load()`
or `!stop_.load(std::memory_order_acquire)` to make it even faster, at all places where `stop_.load()` is called (no performance penalty due to being atomic at all on x86)

Similarly, all places where value is stored into the atomic, should be `stop_.store(value)`, or `stop_.store(value, std::memory_order_release)`.